katello/hooks/post/09-mongo_syspath.rb

```diff
@@ -1,17 +0,0 @@
-def install_syspaths
-  if facts[:os][:release][:major] == '7'
-    `rpm -qa | grep rh-mongodb34-syspaths`
-    if $?.success?
-      logger.debug 'rh-mongodb34-syspaths already installed, skipping.'
-    else
-      logger.info 'rh-mongodb34-syspaths not present, installing.'
-      `yum install -y -q rh-mongodb34-syspaths`
```

Comment:
I think we still need to ensure this is installed. IIRC that can be done by setting the `mongodb::globals::server_package_name` to that in `config/foreman.hiera/family/RedHat-7.yaml`